src/js/hooks/list-pages/inbound/useInboundListTableData.jsx

```diff
@@ -14,24 +14,32 @@ import exportFileFromAPI from 'utils/file-download-util';
 import { translateWithDefaultMessage } from 'utils/Translate';
 
 const useInboundListTableData = (filterParams) => {
-  const messageId = 'react.stockMovement.inbound.fetching.error';
+  const errorMessageId = 'react.stockMovement.inbound.fetching.error';
   const defaultMessage = 'Unable to fetch inbound movements';
```

Comment:
changing from `messageId` to `errorMessageId` as I requested, you should also change the `defaultMessage` naming to `defaultErrorMessage`